e2e/cli/command/provision.go

```diff
@@ -0,0 +1,325 @@
+package command
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+
+	getter "github.com/hashicorp/go-getter"
+	hclog "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad/helper/discover"
+	"github.com/mitchellh/cli"
+)
+
+func init() {
+	getter.Getters["file"].(*getter.FileGetter).Copy = true
+}
+
+func ProvisionCommandFactory(ui cli.Ui, logger hclog.Logger) cli.CommandFactory {
+	return func() (cli.Command, error) {
```

Comment:
Hmm why don't you just take the meta? Currently every command will have its own so if you are potentially redoing work across all commands as more gets added to the meta